contracts/contracts/strategies/MorphoCompoundStrategy.sol

```diff
@@ -190,24 +190,27 @@ contract MorphoCompoundStrategy is BaseCompoundStrategy {
         address _asset,
         uint256 _amount
     ) external override onlyVault nonReentrant {
-        _withdraw(_recipient, _asset, _amount);
+        _withdraw(_recipient, _asset, _amount, false);
     }
 
     function _withdraw(
         address _recipient,
         address _asset,
-        uint256 _amount
+        uint256 _amount,
+        bool _skipAmountCheck
```

Comment:
@DanielVF added this change so when `withdrawAll` is called the call doesn't fail when there are no funds in Morpho Strategy.

Reply:
Hmmm. This feels ugly. What if you do an if check in withdrawAll() and don't call _withdraw unless you have a balance?

Reply:
good idea, cleaner solution